### DISCUSSION TITLE
Hot to set Win3.11

### DISCUSSION THREAD

**Discussion by AP2020C** [joncampbell123/dosbox-x#5350] [Q&A]
I tried reading https://dosbox-x.com/wiki/Guide%3AInstalling-Windows-3.1x and now I can run Win3.11 in DosBox-X!

Questions:
* the resolution is 640x480, what is the maximum resolution supported? What settings should I enter in the conf to automatically scale?
* Audio is enabled, Windows 3.11 makes sounds, but if I launch a game no audio comes out.
* How do I create shortcuts in Main or Games for the games I copied to C?
* What I love about this DosBox is the ability to save the state, but with F11+S and entering a name, I get “Failed to save the current state” every time

**Comment by rderooy** [comment]
- If you install the S3 video drivers version 1.70.04 with the default emulated S3 video card, you can get up to 1600x1200
- Which game? If sound works in Windows, it should also work in Windows applications like Windows games. But note that not all Windows games supported sound. Also be sure to properly install the Windows game, it may need special INI file settings that otherwise are missing.
- Look up how to create Windows 3.1 PIF files
- The save-state feature has lots of limitations, it cannot always save or restore the state.

**Comment by AP2020C** [comment]
- I downloaded win31.qcow2 from https://archive.org/details/win31_20200819. The instructions I found said that I had to pull it out, but I failed to
- These are:
  - Flak Attack (https://archive.org/details/win3_flakattack)
  - Mole's Quest (https://archive.org/details/win3_molesquest)
 
Which in my collection are found in the Wingame folder. When they ran in my real computer they had a soundtrack, and even emulated in archive.org.

- In DosBox Ykhwong's I never had any problems with saving, although I could never run windows

**Comment by rderooy** [comment]
These seem to be simple shareware games based on what I find. They may not have any sound.
https://www.mobygames.com/game/22874/fast-action-paq/

**Comment by AP2020C** [comment]
Maybe they don't have audio the ones from Mobygames, on Archive they have it

**Comment by rderooy** [comment]
Ok, it seems these games need something called "wavemix" to be installed. That is what seems to be used on the archive.org installation of the games.

See https://www.compuphase.com/wavemix.htm

**Reply by AP2020C** [reply]
I downloaded WaveMix and copied WAVMIX16.DLL and WAVEMIX.INI into \WINDOWS\SYSTEM but the audio still can't be heard.

I also tried downloading MixCfg.
I launched MIXCFG16.EXE from the file manager. It opened a window that looked fine.
Trying to close it, however, I got an error: Mixcfg16 caused a general protection error in the MIXCFG16.exe module at location 0001:0936

**Comment by rderooy** [comment]
I don't have time to look at this, but you may want to have a look at how archive.org runs them from dosbox.

Win 3.1 image, including wavemix:
- https://ia601307.us.archive.org/cors_get.php?path=/33/items/emularity_win31/win31.zip

game overlays:
- https://ia801301.us.archive.org/cors_get.php?path=/27/items/win3_flakattack/flakattack.zip
- https://ia801307.us.archive.org/cors_get.php?path=/16/items/win3_molesquest/molesquest.zip
